index.html

```diff
@@ -1,414 +1,355 @@
-            <div class="list">
-                <ul class="list">
-                    <li><p>CANOPY</p></li>
-                    <li><img src="images/counter-1.png" alt="counter1"></li>
-                    <li><p>Back End Dev</p></li>
-                    <li><img src="images/counter-2.png" alt="counter2"></li>
-                    <li><p>2015</p></li>
-                </ul>
-            </div>
-            <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <br>
-            <div class="buttons">
-                <ul class="list">
-                    <li><div class="tag">html</div></li>
-                    <li><div class="tag">css</div></li>
-                    <li><div class="tag">javascript</div></li>
-                </ul>
-                <button>See Project</button>
-            </div>
-        </section>
-    </section>
-    <!-- Card2 for the Desktop screen -->
-    <section class="card-11">
-            <section class="profile-picture for-cards">
-                <img class="card-img" src="images/snapshort-profile-3.png" alt="Snapshot-profile">
-                <img class="card-img1" src="images/card-img3.png" alt="snapshot-profile">
-            </section>
-        <section class="tonic for-cards">
-            <h1>Facebook 360</h1>
-            <div class="list">
-                <ul class="list">
-                    <li><p>FACEBOOK</p></li>
-                    <li><img src="images/counter-1.png" alt="counter1"></li>
-                    <li><p class="stretch">Full Stack Dev</p></li>
-                    <li><img src="images/counter-2.png" alt="counter2"></li>
-                    <li><p>2015</p></li>
-                </ul>
-            </div>
-            <p class="content">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
-            <br>
-            <div class="buttons">
-                <ul class="list">
-                    <li><div class="tag">html</div></li>
-                    <li><div class="tag">Ruby on rails</div></li>
-                    <li><div class="tag">css</div></li>
-                    <li><div class="tag">javascript</div></li>
-                </ul>
-                <button class="active">See Project</button>
-            </div>
-        </section>
+     <!-- header section -->
+      <header>
+        <nav class="site-nav">
+          <a href="#">
+            <img class="my-logo" src="images/my-logo.png" alt="logo" />
+          </a>
+          <a href="#">
+            <img
+              onclick="myFunction()"
+              class="menu"
+              src="images/mymenu.png"
+              alt="union"
+            />
+          </a>
+          <ul>
+            <li><a href="#">Portolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
```

Comment:
- [ ] You can use `max-width` CSS attribute of another responsive design technique to make it better for any screen size.
- [ ] Check the red lines to check what you should improve (not all if you want) but I strongly recommend you trying to make your design more similar to the Figma design you selected.
- [ ] Add the right background when you are seeing your popup active.

Your mobile design | Figma mobile design

![image](https://user-images.githubusercontent.com/60164239/131983510-94d3d69e-5645-46d7-9570-4987c2d5624a.png)